js/yasgui.local.js

```diff
@@ -1509,8 +1509,14 @@ function refreshList(){
 
   examplesDiv.innerHTML = "";
 
+  var search = document.getElementById('searchExamples').value;
+  var tagsSelected = getSelectedTags();
+  var filter = function(elem) {
+    return shouldShowExample(elem, search, tagsSelected);
+  }
```

Comment:
Le nom de la fonction `filter` est trop générique, on ne sait pas ce qu'elle fait.

Un truc du genre `isSearchedOrTaggedExample` serait meilleur.

En général, pour un `array.filter`, j'utilise une fonction `is*` (puisqu'elle est censée renvoyer un booléen).